### PR TITLE
Initial dev env and instructions

### PR DESCRIPTION
**Pull request by odra**
Adds a basic dev env using devcontainers and instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
 # Sunken Kitchen Challenge
+
+## Project Strucutre
```
Comment (PLeVasseur) [inline README.md]:
```suggestion
## Project Structure
```

**Review by PLeVasseur — COMMENTED** [review]
I know so little about this stuff I don't think I'm even qualified to review it :sweat_smile: 

So does this add the ability to start both the Kuksa data broker and a Zenoh router?

I'd assume we still need to add stuff like:

* dev environment for writing Kuksa stuff
* dev environment for writing uProtocol stuff

?

**Comment by odra** [comment]
So the container is quite flexible, they can install or write whatever they want there, (using vscode).

**Review by PLeVasseur — APPROVED** [review]
I'm not super familiar with containers yet, but seems brings up the Kuksa broker and Zenoh router.

From that point of view, LGTM

**Pull request merged**